### PR TITLE
Addition to cover the dictionary feature

### PR DESCRIPTION
**Pull request by fleguellec**

### CODE REVIEW

File: content/md/first-steps/manage-data-quality.md [hunk 1]
```diff
@@ -48,3 +45,36 @@ In the data quality panel of the product edit form, you can see which attribute
...
+## Locales dictionary
+For users with rights to edit locales, it is possible to add, edit or delete words in the dictionary.
+Go to `Settings/Locales/Dictionary`
+
+![Image data quality dictionary](../img/data-quality-dictionary.png)
+:::
+
+Several ways to add words from the `Dictionary` screens
+- For several locales, by clicking on a locale name and using the mass action to add the same word or words for the selected locales
+- For a single  locale, by using the "add words" box
```
Comment (AymericPlanche) [inline content/md/first-steps/manage-data-quality.md:72]:
```suggestion
- For a single locale, by using the "add words" box
```

File: content/md/first-steps/manage-data-quality.md [hunk 2]
```diff
@@ -48,3 +45,36 @@ In the data quality panel of the product edit form, you can see which attribute
...
+
+![Image data quality add words to dictionary](../img/data-quality-add-words-dictionary.png)
+
+:::info
+To add several words at a time, separate them with comas "," or space.
+You can add up to 100 words at a time.
+Words are not case-sensitive: you can either type them all in lower or uppercases.
```
Comment (AymericPlanche) [inline content/md/first-steps/manage-data-quality.md:79]:
```suggestion
Words are case-insensitive: you can either type them all in lower or uppercases.
```

File: content/md/first-steps/manage-data-quality.md [hunk 3]
```diff
@@ -48,3 +45,36 @@ In the data quality panel of the product edit form, you can see which attribute
...
+
+Several ways to add words from the `Dictionary` screens
+- For several locales, by clicking on a locale name and using the mass action to add the same word or words for the selected locales
+- For a single  locale, by using the "add words" box
+
+![Image data quality add words to dictionary](../img/data-quality-add-words-dictionary.png)
+
+:::info
+To add several words at a time, separate them with comas "," or space.
+You can add up to 100 words at a time.
```
Comment (AymericPlanche) [inline content/md/first-steps/manage-data-quality.md:78]: This limit have been removed from the "TagInput" component in the DSM, so we can ignore this rule.

```suggestion
```

**Review by AymericPlanche — APPROVED** [review]

File: content/md/first-steps/manage-data-quality.md [hunk 4]
```diff
@@ -48,3 +45,36 @@ In the data quality panel of the product edit form, you can see which attribute
...
+:::
+
+## Locales dictionary
+For users with rights to edit locales, it is possible to add, edit or delete words in the dictionary.
+Go to `Settings/Locales/Dictionary`
+
+![Image data quality dictionary](../img/data-quality-dictionary.png)
+:::
+
+Several ways to add words from the `Dictionary` screens
```
Comment (NolwennP) [inline content/md/first-steps/manage-data-quality.md:70]:
```suggestion
Several ways to add words from the `Dictionary` screens:
```

File: content/md/first-steps/manage-data-quality.md [hunk 5]
```diff
@@ -48,3 +45,36 @@ In the data quality panel of the product edit form, you can see which attribute
...
+![Image data quality spelling](../img/data-quality-spelling.png)
+:::
+
+:::info
+To save you time, in addition to the manual ways, Akeneo PIM searches all words used in your product data (in text, text area attributes and attribute and option labels. If a word that counts a minimum of 3 letters is found at least 10 times, then this word will be automatically added to your personal dictionary.)
+:::
+
+## Locales dictionary
+For users with rights to edit locales, it is possible to add, edit or delete words in the dictionary.
+Go to `Settings/Locales/Dictionary`
```
Comment (NolwennP) [inline content/md/first-steps/manage-data-quality.md:65]: Shouldn't they need to select a locale first?

File: content/md/first-steps/manage-data-quality.md [hunk 6]
```diff
@@ -48,3 +45,36 @@ In the data quality panel of the product edit form, you can see which attribute
...
+
+Several ways to add words to the `Dictionary`:
+- Through the "ignore" feature in the spell-check pop-in displayed when a possible typo error is detected
+- In the `Locales` settings screen
+
+![Image data quality spelling](../img/data-quality-spelling.png)
+:::
+
+:::info
+To save you time, in addition to the manual ways, Akeneo PIM searches all words used in your product data (in text, text area attributes and attribute and option labels. If a word that counts a minimum of 3 letters is found at least 10 times, then this word will be automatically added to your personal dictionary.)
```
Comment (NolwennP) [inline content/md/first-steps/manage-data-quality.md:60]:
```suggestion
To save you time, in addition to the manual ways, Akeneo PIM searches all words used in your product data (in text, text area attributes and attribute and option labels). If a word that counts a minimum of 3 letters is found at least 10 times, then this word will be automatically added to your personal dictionary.
```

File: content/md/first-steps/manage-data-quality.md [hunk 7]
```diff
@@ -48,3 +45,36 @@ In the data quality panel of the product edit form, you can see which attribute
 
 ![Image data quality attribute groups reminder](../img/data-quality-attribute-groups-reminder.png)
 :::
+
+# Manage your dictionary
+Either because your product names are unique or due to industry acronyms that are not part of regular dictionaries, you can add words in your personal dictionary to fit your special needs and avoid miscalculation of your Quality Score.
```
Comment (NolwennP) [inline content/md/first-steps/manage-data-quality.md:50]: Can we add a link on the "Quality score" words to link to the article that talks about it?

File: content/md/first-steps/manage-data-quality.md [hunk 8]
```diff
@@ -48,3 +45,36 @@ In the data quality panel of the product edit form, you can see which attribute
 
 ![Image data quality attribute groups reminder](../img/data-quality-attribute-groups-reminder.png)
 :::
+
+# Manage your dictionary
```
Comment (NolwennP) [inline content/md/first-steps/manage-data-quality.md:49]: Aren't they multiple dictionaries? One per locale? If yes, then:
```suggestion
# Manage your dictionaries
```

**Pull request merged**